src/App.js

```diff
@@ -58,13 +58,18 @@ export const App = () => {
   }
 
   const handleSkipButton = (event) => {
-      /*if (target.id=="next"){
-          console.log(currentSong)
-          console.log(songList.indexOf(currentSong))
-          setCurrentSong(songList[songList.indexOf(currentSong)+1])
-      } else if (target.id=="back"){
-          setCurrentSong(songList[songList.indexOf(currentSong)-1])
-      } */
+    const currentIndex = currentPlaylist.songs.indexOf(currentSongObject)
+    console.log(currentPlaylist.songs.length)
+      if (event.target.id=="next" && currentIndex<currentPlaylist.songs.length-1){
```

Comment:
this should works like "Spotify"; when you are in the first song and you press " you must to move into the last song; if you are in the last song and press "next", you must to move into the first song

Reply:
ok